code/TagField.php

```diff
@@ -310,7 +310,10 @@ protected function getTags($term) {
 		$term = Convert::raw2sql($term);
 
 		$query = $dataClass::get()
-			->filter($titleField . ':PartialMatch:nocase', $term)
+			->filter(array(
```

Comment:
My understanding is that HasManyList should already include this filter.

Also, you run the risk where $source is actually a DataList, not a *ManyList, and thus won't have a method `getForeignKey`.

Did you encounter this issue where tags / categories from other blogs were actually being included?